scala-package/examples/pom.xml

```diff
@@ -14,6 +14,24 @@
   <name>MXNet Scala Package - Examples</name>
 
   <profiles>
+    <profile>
```

Comment:
What happens if an undefined platform is being detected?

Reply:
Only the mentioned platforms are supported similar to what the current scala-package supports.